test/unit/search/fm_index_cursor/bi_fm_index_cursor_collection_test_template.hpp

```diff
@@ -61,41 +54,39 @@ TYPED_TEST_P(bi_fm_index_cursor_collection_test, extend)
 
 TYPED_TEST_P(bi_fm_index_cursor_collection_test, extend_char)
 {
-    std::vector<seqan3::dna4_vector> text{"ACGGTAGGACG"_dna4, "TGCTACGATCC"_dna4};
-    typename TypeParam::index_type bi_fm{text};
+    typename TypeParam::index_type bi_fm{this->text_col2};  // {"ACGGTAGGACG", "TGCTACGATCC"}
 
     auto it = bi_fm.begin();
-    EXPECT_TRUE(it.extend_left('G'_dna4)); // "G"
+    EXPECT_TRUE(it.extend_left(this->text[2])); // "G"
     EXPECT_EQ(seqan3::uniquify(it.locate()), (std::vector<std::pair<uint64_t, uint64_t>>{{0, 2}, {0, 3}, {0, 6},
                                                                                          {0, 7}, {0, 10},
                                                                                          {1, 1}, {1, 6}}));
-    EXPECT_TRUE(it.extend_left('C'_dna4)); // "CG"
+    EXPECT_TRUE(it.extend_left(this->text[1])); // "CG"
     EXPECT_EQ(seqan3::uniquify(it.locate()), (std::vector<std::pair<uint64_t, uint64_t>>{{0, 1}, {0, 9}, {1, 5}}));
-    EXPECT_FALSE(it.extend_left('C'_dna4)); // "CG"
+    EXPECT_FALSE(it.extend_left(this->text[1])); // "CG"
```

Comment:
Shouldn't the string in the comment be "CCG" here? 

If so, then the comments in 68,70,72,74,76,78,80 should be updated too I believe.